dist/index.html

```diff
@@ -31,22 +31,14 @@ <h3>Recent Scores</h3>
                         <span class="load"></span>
                     </button>
                 </div>
-                <ul class="ul">
-                    <li>Name: 100</li>
-                    <li>Name: 100</li>
-                    <li>Name: 100</li>
-                    <li>Name: 100</li>
-                    <li>Name: 100</li>
-                    <li>Name: 100</li>
-                    <li>Name: 100</li>
-                </ul>
+                <ul class="ul"></ul>
```

Comment:
**[OPTIONAL BUT HIGHLY RECOMMENDED]**

- [ ] It would be nice if you make your own logo of your name and put that in the readme file at the top to present yourself as a brand in the job market or tech industry to catch the recruiters' eyes 👍